doc/user/OPTIONS.md

```diff
@@ -29,66 +29,66 @@ F3D behavior can be fully controlled from the command line using the following o
+| \-\-hdri-file=\<HDRI file\>                 | string<br>-      | Set the _HDRI_ image that can be used as ambient lighting and skybox.<br>Valid file format are hdr, exr, png, jpg, pnm, tiff, bmp. <br> If not set, a default is provided.                                                                                                                                                               |
+| \-\-hdri-ambient                            | string<br>-      | Light the scene using the _HDRI_ image as ambient lighting.<br>The environment act as a light source and is reflected on the material.                                                                                                                                                                                                   |
+| \-\-texture-matcap=\<texture file\>         | string<br>-      | Set the texture file to control the material capture of the object. All other model options for surfaces are ignored if this is set. Must be in linear color space. <br>Model specified by default.                                                                                                                                      |
+| \-\-texture-base-color=\<texture file\>     | string<br>-      | Set the texture file to control the color of the object. Please note this will be multiplied with the color and opacity options. Must be in sRGB color space. <br>Model specified by default.                                                                                                                                            |
+| \-\-texture-material=\<texture file\>       | string<br>-      | Set the texture file to control the occlusion, roughness and metallic values of the object. Please note this will be multiplied with the roughness and metallic options, which have impactful default values. To obtain true results, use \-\-roughness=1 \-\-metallic=1. Must be in linear color space. <br>Model specified by default. |
+| \-\-texture-emissive=\<texture file\>       | string<br>-      | Set the texture file to control the emitted light of the object. Please note this will be multiplied with the emissive factor. Must be in sRGB color space. <br>Model specified by default.                                                                                                                                              |
+| \-\-emissive-factor=\<R,G,B\>               | color<br>-       | Set the emissive factor. This value is multiplied with the emissive color when an emissive texture is present. <br>Model specified by default.                                                                                                                                                                                           |
```

Comment:
Should we replace `<R,G,B>` by just `<color>` since there will be multiple way to parse colors?